base/cvd/fastboot/fs.cpp

```diff
@@ -0,0 +1,247 @@
+#include "fs.h"
```

Comment:
nit: the original file doesn't have a license, should this file have one?

Reply:
Done